data/src/main/java/org/oppia/data/persistence/PersistentCacheStore.kt

```diff
@@ -92,6 +95,32 @@ class PersistentCacheStore<T : MessageLite> private constructor(
     }
   }
 
+  /**
+   * Callers should use this read function if they they don't care or specifically do not
+   * want to observe changes to the underlying store. If the file is not in memory, it will
+   * loaded from disk and observers will be notified.
+   *
+   * @return a deferred value that contains the value of the cached payload.
+   */
+  // Returns a deferred value that contains the cache payload value
```

Comment:
Can we remove this now that there's a KDoc saying the same thing?

Reply:
Removed.